get_ioc_usage.py

```diff
@@ -32,7 +33,17 @@ def calc_iocs_on_intruments(instruments):
             iocs_in_component = set(component_utils.get_iocs(component_utils.get_iocs_xml(component)))
             instrument_configs[instrument_name] = instrument_configs[instrument_name].union(iocs_in_component)
 
-    return instrument_configs
+        device_utils = DeviceUtils(Settings.config_repo_path)
+        try:
+            device_screens = device_utils.get_device_screens(device_utils.get_device_screens_from_xml())
+        except Exception:
+            # Instrument does not have device screen directory
+            device_screens = []
+
+        instrument_name = instrument["name"]
```

Comment:
I think `instrument_name` already exists as it's created above - might be able to remove this line?